boards/arm/secureiot1702/doc/secureiot1702.rst

```diff
@@ -0,0 +1,66 @@
+.. _secureiot1702:
+
+Microchip SecureIoT1702
+#######################
+
+Overview
+********
+
+Board configuration for Microchip CEC1702 demo board. This is a cryptographic
+embedded controlled based on ARM Cortex-M4.
+
+Highlights of the board:
+- CEC1702 32-bit ARM® Cortex®-M4F Controller with Integrated Crypto
+- Compact, high-contrast, serial graphic LCD Display Module with back-light
+- 2x4 matrix of push buttons inputs
+- USB-UART Converter as debug interface
+- Potentiometer to ADC channel
+- Serial Quad I/O (SQI) flash
+- OTP programmable in CEC1702
+- Two expansion headers compatible with MikroElektronika mikroBUS™ Expansion interface
+
+
+Supported Features
+==================
+
+Following devices are supported:
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Serial Ports (NS16550)
+- Quad Master-only SPI controller (QMSPI)
+
+
+Connections and IOs
+===================
+
+Please refer to the demo board schematics for the pin routings. Additional
+devices can be connected via mikroBUS expansion interface.
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+[How to use this board with Zephyr and how to flash a Zephyr binary on this
+device]
+
+
+Debugging
+=========
+[ How to debug this board]
+
+
+References
+**********
+
```

Comment:
These target link definitions won't show up here in the generated output.  If you'd like them to appear, you can add a directive here:

    .. target_notes::
         
(be sure there's a blank line after this directive).  If you don't want these links to show up here in the output, then delete the ``References`` heading above (and don't put in the target_notes directive :)